content/en/solutions/web/typescript/solution 1/tsconfig.json

```
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "outDir": "dist",
+    "strict": true
+  },
+  "include": [
+    "solution 1/src/**/*.ts"
```

Diese Datei lässt den Build failen... Ich weiss nicht was der Check genau macht, aber für mich macht dieses `tsconfig.json` so mehr Sinn:

```suggestion
    "./src/**/*.ts"
```